frontend/views/containers/proposals/ProposalTemplate.vue

```diff
@@ -25,7 +25,10 @@
 
       .c-confirmation(v-if='isConfirmation' key='confirmation')
         svg-proposal.c-svg
-        i18n(html='Members of your group will now be asked to vote.</br>You need <strong>{value} yes votes</strong> for  your proposal to be accepted.' :args='{value: rule.value}')
+        i18n(
+          compile
+          :args='{ ...LTags("strong"), value: rule.value }'
+        ) Members of your group will now be asked to vote.{br_} You need {strong_}{value} yes votes{_strong} for your proposal to be accepted.
```

Comment:
- `compile` isn't needed here because this doesn't contain any Vue.js components
- Can you rename `value` to to `numVotes` to give translators more context?